kythe/release/release.BUILD

```diff
@@ -11,14 +11,28 @@ exports_files([
 load(":extractors.bzl", "extractor_action")
 load(":vnames.bzl", "construct_vnames_config")
 
+config_setting(
+    name = "assign_external_projects_to_separate_corpora",
+    values = {
+        "define": "kythe_assign_external_projects_to_separate_corpora=true",
+    },
+)
+
 construct_vnames_config(
     name = "vnames_config",
-    srcs = [
-        "vnames.cxx.json",
-        "vnames.go.json",
-        "vnames.java.json",
-        "vnames.json",
-    ],
+    srcs = select({
+        "//conditions:default": [
+            # by default, the simple vname rules are used, which map everything
```

Comment:
The defaults feel a little weird.  It seems like we actually default to the other branch because kythe_assign_external_projects_to_separate_corpora is defaulted to true.

Maybe switch this around?  (Make the `"//conditions:default"` be the actual default of the preexisting behavior and then have a new `kythe_singleton_corpus` variable that defaults to false and trigger the simple_vnames.json usage.)

Reply:
kythe_assign_external_projects_to_separate_corpora should be defaulted to false/unset.

I think most people that use this will want the single-corpus behavior, so I'd have a slight preference for that behavior being the default if no extra flags are specified. Thoughts?

If you think a `kythe_singleton_corpus` flag is better, I'm fine with changing to that.